entrypoint.sh

```diff
@@ -477,17 +477,19 @@ appManagePy() {
 }
 appBackup() {
     echo "Starting backup process ..."
-    if [ -d "/tmp/backup-$(date "+%D-%H-%M-%S")" ]; then
-        echo "Temporary backup folder for \"$(date "+%D-%H-%M-%S")\" already exists. Aborting."
+    local TIMESTAMP
+    TIMESTAMP=$(TZ=UTC date "+%FT%TZ")
```

Comment:
Looks like a good iterative improvement to me -- thanks for thinking about Windows host compatibility!

Go ahead and squash such a change in (adding a `Co-authored-by`, and updating the commit message with your above explanation) and I'll gleefully push the "merge" button.

This date fix becomes irrelevant when we switch to `./manage.py backup`, so it's not worth iterating _too_ much on, particularly since it has (AFAICT) never actually worked in the past because of the `/`s in the path -- but making it at least _functional_ seems like a fine intermediate state to land at. :)